eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventlist/view/EventListToolbarImpl.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.eventlist.view;
+
+import android.content.res.Resources;
+import android.support.v4.app.FragmentActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.schedjoules.eventdiscovery.R;
+import com.schedjoules.eventdiscovery.common.BaseActivity;
+
+
+/**
+ * The toolbar of the Event list screen.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class EventListToolbarImpl implements EventListToolbar
```

Comment:
What's this class all about? What does it add? Could the initialization go to a method instead? The `setToolbarTitle` method doesn't seem to add anything.
Is this really a class?